website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,100 @@
+## Usage
+
+### When to use
+
+- When displaying code examples and longer snippets of code that benefit from syntax highlighting.
+
+### When not to use
+
+- As a full-blown code editor
+- When displaying keys, tokens, or other generated values, use a [CopySnippet](/components/copy/snippet) instead.
```

Comment:
https://github.com/hashicorp/design-system/pull/1691/files#r1367333392

Reply:
Resolved in https://github.com/hashicorp/design-system/pull/1749/commits/19a092530be5f2419739f829a8d000a99a7cb4a7

Reply:
I also added a section for when to use the CopySnippet vs CodeBlock, trying to think of more examples, what do you think?

Reply:
I'll add a comment there. In the meantime I'm resolving this one.